(libreoffice) Clarify package update lagging behind software release and where to raise issues / questions

pauby opened the pull request:
## Description
I've updated the README*.md files for libreoffice. These should flow into the 'description` in the .nupspec file.

Note that how we add this info, the message and the formatting are all up for debate. I wanted to get something up here for discussion. Once agreed, I feel we should require any updates to a package to add this to the README files (perhaps by adding it to #1754).

Closes #1803.

## Motivation and Context

Questions are still being asked via Disqus for many packages. As Disqus is generally not responded to it makes sense to clarify where questions and issues should be raised. Going forward we should add this to all packages.

## How Has this Been Tested?
I haven't tested this as it was an update to Markdown.

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:
- [ ] My code follows the code style of this repository.
- [x] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

pauby commented:
This looks to be a build automation issue.

![image](https://user-images.githubusercontent.com/12760779/154917998-ce94c89e-052b-4a77-96f4-6172d58f9c50.png)

AdmiringWorm commented on automatic/libreoffice-streams/README.fresh.md:
```diff
@@ -6,3 +6,5 @@ LibreOffice is the free power-packed Open Source personal productivity suite for
 - This package installs LibreOffice Fresh which is the stable version of LibreOffice with the most recent bleeding-edge features. This version is recommended for technology enthusiasts, early adopters or power users. For more conservative users or for deployments in enterprise or corporate environments, it is recommended to install [libreoffice-still](/packages/libreoffice-still) instead.
 - Older versions of this package are likely broken because the LibreOffice team removes the download links for older versions after each update.
+
+**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/ ...
```
```suggestion
**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to.**
```

Otherwise it won't show up correctly as bold text.

AdmiringWorm commented on automatic/libreoffice-streams/README.md:
```diff
@@ -6,3 +6,5 @@ LibreOffice is the free power-packed Open Source personal productivity suite for
 - This package installs LibreOffice Fresh which is the stable version of LibreOffice with the most recent bleeding-edge features. This version is recommended for technology enthusiasts, early adopters or power users. For more conservative users or for deployments in enterprise or corporate environments, it is recommended to install [libreoffice-still](/packages/libreoffice-still) instead.
 - Older versions of this package are likely broken because the LibreOffice team removes the download links for older versions after each update.
+
+**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/ ...
```
```suggestion
**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to.**
```

Otherwise it won't show up correctly as bold text.

AdmiringWorm commented on automatic/libreoffice-streams/README.still.md:
```diff
@@ -7,3 +7,5 @@ LibreOffice is the free power-packed Open Source personal productivity suite for
 - For business deployments, The Document Foundation (the foundation behind the LibreOffice project) strongly recommends [support from certified partners](https://www.libreoffice.org/get-help/professional-support/) which also offer long-term support versions of LibreOffice.
 - Older versions of this package are likely broken because the LibreOffice team removes the download links for older versions after each update.
+
+**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/ ...
```
```suggestion
**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to.**
```

Otherwise it won't show up correctly as bold text.

AdmiringWorm reviewed (changes requested):
Just some formatting issues that needs to be fixed, otherwise this looks good to me.

pauby commented:
Thanks @AdmiringWorm. I'll get that space removed (can't believe I didn't see it! 😄 ). I'll fixing up the other PR's too as they will likely have the same issue.

AdmiringWorm commented:
@pauby I have fixed it up on the other PR's you made, I didn't notice the space until after I had merged those.

pauby commented:
Updated this PR with those changes.

> @pauby I have fixed it up on the other PR's you made, I didn't notice the space until after I had merged those.

If it's not going to render properly in those other packages, do you want me to fix them?

pauby commented:
@AdmiringWorm Going to put this into Draft just now as I may as well add [this](https://github.com/chocolatey-community/chocolatey-packages/issues/1803#issuecomment-1047273752) to it while I'm here.

pauby commented:
@AdmiringWorm This is ready for review again.

AdmiringWorm reviewed (approved):
LGTM

AdmiringWorm commented:
@pauby your changes have been merged, thanks for your contribution 👍
